Apps/W1/Shopify/app/src/Order handling/Tables/ShpfyOrderLineAttribute.Table.al

```diff
@@ -0,0 +1,40 @@
+/// <summary>
+/// Table Shpfy Order Line Attribute (ID 30123).
+/// </summary>
+table 30148 "Shpfy Order Line Attribute"
```

Comment:
The default value for Shopify Objects is Access = Internal and extensible = false.
If you need some table public, then please outline the scenario for discussion with microsoft.

Reply:
ok, added properties